test/sql/debug_helpers.sql

```diff
@@ -1,8 +1,9 @@
-
 CREATE EXTENSION IF NOT EXISTS vector;
 CREATE EXTENSION IF NOT EXISTS lanterndb;
```

Comment:
I think `pg_regress` has an option to create the extension when it is invoked so we do not have to add these in all files.

Reply:
It has an option `--load-extension=` , but if I was loading the extension like this, I was getting error that prop `hnsw.init_k` does not exists. I will try to debug and make this work.